### PR TITLE
fishops nerf real

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
made rehydratables inherit forensic fibers + dna + prints when hydrated

fibers and dna makes sense prints are a stretch but magic forensic analyser thingy whatever

## Why / Balance
"your honour they have no evidence" in fishops trial no longer works
det with 10 separate fish scans that all have your prints on them:

now you have to cover fingerprints like normal or butcher all the fish to avoid being sentenced

## Technical details
moved rehydrating to shared for future stuff, also fixed the rehydrate popup because did you know it used to have one

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
rehydrating is in shared so if you use `GotRehydratedEvent` import `Content.Shared.Chemistry.Components;` (instead of) server components.

**Changelog**
:cl:
- tweak: Rehydrated mobs (carp) now keep their forensics data from their cube/plushie form.

### CODE REVIEW

**Comment by IProduceWidgets** [comment]
In b4 petting doesn't work unless you take your gloves off PR image

**Comment by deltanedas** [comment]
test fail fake

**Comment by deltanedas** [comment]
merg

File: Content.Server/Forensics/Systems/ForensicsSystem.cs [hunk 1]
```diff
@@ -71,6 +73,25 @@ private void OnMeleeHit(EntityUid uid, ForensicsComponent component, MeleeHitEve
...
+            {
+                target.Fibers.Add(fiber);
+            }
+
+            foreach (var print in ent.Comp.Fingerprints)
+            {
+                target.Fingerprints.Add(print);
+            }
+        }
+
```
Comment (metalgearsloth) [inline Content.Server/Forensics/Systems/ForensicsSystem.cs:94]: Just add a method for this.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Comment by deltanedas** [comment]
@metalgearsloth merg

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**